src/main/webapp/WEB-INF/templates/index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+<head>
+
+    <title>Quick Start - Leaflet</title>
+
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="shortcut icon" type="image/x-icon" href="docs/images/favicon.ico"/>
+
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.1/dist/leaflet.css"/>
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+
+    <script src="https://unpkg.com/leaflet@1.0.1/dist/leaflet.js"></script>
+
+    <script src="scripts/events.js" type="text/javascript"></script>
+
+    <style>
+        #mapid {
+            width: 1200px;
+            height: 700px;
+        }
+    </style>
+</head>
+<body>
+
+<div class="container" style="width:80%">
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-sm-12 header" id="header">
+                <h1 id="logo">
+                    <img id="logo" class="logo" src="logo/Logo/W6.png"/>
+                </h1>
+                <h2 id="site-name">W6 assess</h2>
+                <h4 id="site-slogan">news based security profiling</h4>
+                <br><br>
+                <nav class="navbar navbar-inverse" style="background-color:black; width:106%">
+
+                    <ul class="nav navbar-nav">
+                        <li class="active"><a href="#">Home</a></li>
+                        <li><a href="input">Input</a></li>
+                        <li><a href="#">Report</a></li>
+                        <li><a href="#">Add Event</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+    </div>
+    <div class="" id="main-title">
+        <h1 style="color:#39B7CD">Map Visualization</h1>
+
+    </div>
+
+    <div class="row">
+        <div class="col-xs-12  index-main-row">
+            <div class="row row-index">
+
+                <div class="col-lg-6  col-md-3 col-xs-12 col-md-offset-0 index-img-client">
+
+                    <p> Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
```

Comment:
I don't know. That's in your branch:
https://github.com/W6-Assess/dev/blob/map/src/main/webapp/WEB-INF/jsp/index.jsp